core/base/Dockerfile

```diff
@@ -61,16 +61,32 @@ RUN set -euxo pipefail \
         mkdir -p /root/.cargo/registry/index && \
         git clone --bare https://github.com/rust-lang/crates.io-index.git /root/.cargo/registry/index/github.com-1285ae84e5963aae \
     ; pip install -r requirements-${MAILU_DEPS}.txt \
-    ; apk del -r .build-deps \
```

Comment:
The build deps need to be deleted before the end of the RUN command. Please don't remove this step.
If you need something in a later step there should be two apk add commands and two virtual's. The first `apk add` has to be in line 53 then.

Reply:
Done in 63a12d98